app/celery/tasks.py

```diff
@@ -803,3 +806,28 @@ def acknowledge_receipt(notification_type: Any, process_type: Any, receipt: UUID
         return
     else:
         current_app.logger.error(f"acknowledge_receipt: receipt {receipt} not found in any queue")
+
+
+@notify_celery.task(name="seed-bounce-rate-in-redis")
+@statsd(namespace="tasks")
+def seed_bounce_rate_in_redis(service_id: str, interval: int = 24):
```

Comment:
I guess this is going to be called from `app/celery/scheduled_tasks.py` in a future PR?

Reply:
I was thinking about this and added in the latest commit that will initialize the seeded data